Source/PubnubLibrary/Private/PubnubSubsystem.cpp

```diff
@@ -902,7 +866,8 @@ bool UPubnubSubsystem::CheckIsPubnubInitialized()
 {
 	if(!IsInitialized)
 	{
-		PubnubError("Pubnub is not initialized. Aborting operation");
+		PubnubError("Pubnub is not initialized correctly. Aborting operation. Make sure to call InitPubnub or "
+			"set InitializeAutomatically to true in Pubnub SDK settings. Also make sure that publish and subscribe keys are correct.");
```

Comment:
is it possible to detect what exactly failed?

Reply:
In most cases init wasn't called, that's why it's pointed in the message.